javascript.js

```diff
@@ -1,89 +1,99 @@
+    this.updateButtonState = this.updateButtonState.bind(this);
+    this.renderBooks = this.renderBooks.bind(this);
+
+    this.bookAddition.addEventListener('click', this.addBook);
+    this.titleInput.addEventListener('input', this.updateButtonState);
+    this.authorInput.addEventListener('input', this.updateButtonState);
+
+    this.renderBooks();
+
+    const storedTitle = window.localStorage.getItem('title');
+    const storedAuthor = window.localStorage.getItem('author');
+    if (storedTitle) {
+      this.titleInput.value = storedTitle;
+    }
+    if (storedAuthor) {
+      this.authorInput.value = storedAuthor;
+    }
 
-function addBook() {
-  function updateButtonState() {
-    if (!titleInput.value || !authorInput.value || titleInput.value.trim() === '' || authorInput.value.trim() === '') {
-      bookAddition.disabled = true;
+    this.titleInput.addEventListener('input', () => {
+      window.localStorage.setItem('title', this.titleInput.value);
+    });
+    this.authorInput.addEventListener('input', () => {
+      window.localStorage.setItem('author', this.authorInput.value);
+    });
+  }
+
+  updateButtonState() {
+    if (!this.titleInput.value || !this.authorInput.value || this.titleInput.value.trim() === '' || this.authorInput.value.trim() === '') {
+      this.bookAddition.disabled = true;
     } else {
-      bookAddition.disabled = false;
+      this.bookAddition.disabled = false;
     }
   }
 
-  function renderBooks() {
-    dynamiclist.innerHTML = '';
+  renderBooks() {
+    this.dynamicList.innerHTML = '';
 
     const storedBooks = JSON.parse(window.localStorage.getItem('books'));
     if (storedBooks) {
-      books = storedBooks;
-      books.forEach((book) => {
+      this.books = storedBooks;
+      this.books.forEach((book, i) => {
         const bookEntry = document.createElement('div');
+        if (i % 2 !== 0) {
+          bookEntry.style.backgroundColor = 'lightgrey';
+        }
         bookEntry.innerHTML = `
-          <p>${book.author}</p>
-          <p>${book.title}</p>
+          <p>"${book.title}" by ${book.author}</p>
           <button class="removebtn">Remove</button>
-          <span></span>
         `;
 
         const removeBtn = bookEntry.querySelector('.removebtn');
         removeBtn.addEventListener('click', () => {
           const { parentElement } = removeBtn;
           parentElement.remove();
-          // Update the books array after removing
-          books = books.filter((b) => b.title !== book.title && b.author !== book.author);
-          // Update the local storage with the updated books array
-          window.localStorage.setItem('books', JSON.stringify(books));
+          this.books = this.books.filter((b) => b.title !== book.title && b.author !== book.author);
+          window.localStorage.setItem('books', JSON.stringify(this.books));
         });
 
-        dynamiclist.appendChild(bookEntry);
+        this.dynamicList.appendChild(bookEntry);
       });
     }
   }
```

Comment:
Kindly add a separate method to remove books. It would take `bookId` as a parameter and remove the book accordingly from the list.
![image](https://github.com/agneta1991/awsome-books/assets/33268927/338bf009-e24e-4c8b-8f0f-525043d5c3d1)